specification/eventhub/resource-manager/readme.md

```diff
@@ -206,6 +206,27 @@ input-file:
 - Microsoft.EventHub/preview/2023-01-01-preview/ApplicationGroups.json
 ```
 
+### Tag: package-2024-01
+
+These settings apply only when `--tag=package-2024-01` is specified on the command line.
+
+``` yaml $(tag) == 'package-2024-01'
+input-file:
+- Microsoft.EventHub/stable/2024-01-01/AvailableClusterRegions.json
+- Microsoft.EventHub/stable/2024-01-01/Clusters.json
+- Microsoft.EventHub/stable/2024-01-01/namespaces.json
+- Microsoft.EventHub/stable/2024-01-01/quotaConfiguration.json
+- Microsoft.EventHub/stable/2024-01-01/networkrulessets.json
+- Microsoft.EventHub/stable/2024-01-01/AuthorizationRules.json
+- Microsoft.EventHub/stable/2024-01-01/CheckNameAvailability.json
+- Microsoft.EventHub/stable/2024-01-01/consumergroups.json
+- Microsoft.EventHub/stable/2024-01-01/disasterRecoveryConfigs.json
+- Microsoft.EventHub/stable/2024-01-01/operations.json
+- Microsoft.EventHub/stable/2024-01-01/eventhubs.json
+- Microsoft.EventHub/stable/2024-01-01/SchemaRegistry.json
+- Microsoft.EventHub/stable/2024-01-01/ApplicationGroups.json
+```
+
 ## Suppression
```

Comment:
suppressions:
  - code:  OperationIdNounVerb
    from: AvailableClusterRegions.json
    where: $.paths./subscriptions/{subscriptionId}/providers/Microsoft.EventHub/availableClusterRegions.get
    reason: Fixing it would be SDK breaking